src/components/AdImage/AdImage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { AdImageProps } from '@/types/AdImage';
+const baseURL = import.meta.env.BASE_URL;
+
+const AdImage: React.FC<AdImageProps> = ({ src, alt, href, className = '' }) => {
+  return (
+    <a href={href} className="block p-4">
```

Comment:
Ya que es una imagen, debería poner un aria label

Reply:
First off, I just wanted to express my immense gratitude for your suggestion to replace the alt attribute with an aria-label for the image. I mean, who knew that the super-detailed and entirely comprehensive alt text — "Sale. Get up to 30% off all home products. Buy now" — just wasn't up to the task?

Clearly, those five highly specific words, brimming with more information than a Wikipedia article, couldn't possibly suffice for screen readers, and we really should be crafting an aria-label to tell them exactly what they already know.

Because who needs brevity and simplicity when we could double up on accessibility attributes and get our point across twice as hard, am I right? Imagine the auditory feast as a screen reader lovingly recites my verbose alt text, only to immediately follow it up with an aria-label that says... well... the same thing. It’s kind of like having two waiters at a restaurant explain the specials at the same time – one from your left ear, the other from your right. It's a delightful cacophony of redundancy.

But seriously, thank you for the tip! Next time I’ll just make sure to add an aria-label saying “Close your eyes and imagine a banner about home products because this descriptive alt text wasn’t quite descriptive enough.”

Can't wait for your next review — it's always a learning experience!